### ISSUE TITLE
Crashing on Real Device after Build

### ISSUE THREAD

**Issue by mono424** [ArjanAswal/Stockfish#8]
Well sorry for posting so many issues but in this early stage it will okay I guess.
So on Android and iOS Emulator everything works well. But after a APK-Build on a real device the Application crashes as soon as I initialize stockfish. Cant really tell if its the initialization or some command run in the beginning as its the release version.
Do you have seen such issues probably already?

Best Regards
Khadim

**Comment by daohoangson** [comment]
> Error changing dalvik-cache ownership : Permission denied

Sounds serious. I have just built the example app in release mode and it ran fine on an Android device... The crashing app your app or are you testing with the example one?

**Comment by mono424** [comment]
I am runnning my app. Probably i should try the example one on the device .)

**Comment by daohoangson** [comment]
Yes, please do that. Also, are you running Android 12 beta on the Pixel 5?

**Comment by mono424** [comment]
I tested now on android 11 and it works. Do you have any ideas how to get it running on android 12 aswell ? :)

**Comment by daohoangson** [comment]
No idea yet, probably related to the old NDK version though...

**Comment by daohoangson** [comment]
I have just investigated NDK versioning (see https://github.com/ArjanAswal/Stockfish/issues/7#issuecomment-850581244). Can you try bumping gradle to 4.2.0 as suggested in that comment in order to use NDK 21.4? Newer version may have better compatibility with Android 12.

**Comment by mono424** [comment]
Thank you really much will try that as soon as i can. Probably Sunday :)

**Comment by mono424** [comment]
I tried out, bumped gradle to 4.2.0. Still crashing.. probably not very bad cause its only android 12 but still a pitty :/
